advanced_functionality/autogluon-multimodal-containers/scripts/multimodal_serve.py

```diff
@@ -0,0 +1,60 @@
+import base64
+import hashlib
+import json
+from io import BytesIO, StringIO
+
+import pandas as pd
+from PIL import Image
+
+from autogluon.multimodal import MultiModalPredictor
+
+
+def _decode_and_save_base85_image(image_b85):
+    """saves Base65 encoded image to disk and returns unique name"""
+
+    image_bytes = base64.b85decode(image_b85)
+    image_hash = hashlib.md5(image_bytes).hexdigest()
+    image_name = f"image_{image_hash}.png"
+    image = Image.open(BytesIO(image_bytes))
+    image.save(image_name)
+
+    return image_name
+
+
+def model_fn(model_dir):
+    """loads model from previously saved artifact"""
+
+    model = MultiModalPredictor.load(model_dir)
+    globals()["column_names"] = [
+        column_name
+        for column_name in model._column_types.keys()
+        if column_name != model._label_column
+    ]
+
+    return model
+
+
+def transform_fn(model, request_body, input_content_type, output_content_type="application/json"):
+
+    if input_content_type != "text/csv":
```

Comment:
Is it the right content_type for image job?

Reply:
In this tutorial I chose to encode the images as Base85 ASCII in the DataFrame and serialize the DataFrame as CSV on the client side before making the request to the endpoint. If I had chosen a different method for encoding and serializing the images, the `input_content_type` would need to change. But based on that decision, I believe "text/csv" is correct.